rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -593,6 +594,18 @@ upgrade_slapdconf() {
     fi
   done
 
+  # 6.2.0~beta2:
+  # Add software index in slapd.conf
+  SLAPD_CONF="/opt/rudder/etc/openldap/slapd.conf"
+  NEED_REINDEX=false
+  line="index\s*software\s*eq"
+  INDEX_COUNT=$(grep -c "^${line}" ${SLAPD_CONF} 2>/dev/null || true)
+  if [ ${INDEX_COUNT} -eq 0 ]; then
+    echo "Adding LDAP index on attribute: software"
+    sed -i "/index\s*objectClass\s*eq/a index\tsoftware\teq" ${SLAPD_CONF}
+    NEED_REINDEX=true
```

Comment:
you need to change also the logic below to check for the new index as well

Reply:
i would simply 

      INDEX_BKP_COUNT=$(grep -c "^index\s*software\s*eq" ${SLAPD_CONF_BKP} 2>/dev/null || true)
      if [ ${INDEX_BKP_COUNT} -eq 0 ]; then
        NEED_REINDEX=true
      fi